recording/docs/installation.md

```diff
@@ -0,0 +1,69 @@
+# Installation
+
+The recording server can be installed using system packages in some GNU/Linux distributions. A "manual" installation is required for others.
+
+## System setup
+
+Independently of the installation method, the recording server requires some dependencies that are not typically found in server machines, like Firefox. It is highly recommended to setup a firewall that prevents any access from the outside to the machine, except those strictly needed by the recording server (and, of course, any additional service that might be needed in the machine, like SSH).
+
+The recording server acts similar to a regular participant in the call, so the firewall needs to allow access to the Nextcloud server and the HPB. These are the connections that need to be allowed from the recording server:
+- Nextcloud server using HTTPS (TCP on port 443 of the Nextcloud server)
+- HPB using HTTPS (TCP on port 443 of the signaling server).
+  The HTTPS connection must be upgradeable to a WebSocket connection.
+- HPB using UDP.
+  The recording server connects to a port in the range 20000-40000 (or whatever range is configured in Janus, the WebRTC gateway), while the WebRTC gateway may connect on any port of the recording server.
+
+Both the HPB and the recording server should be in the same internal network, although preferably not in the same machine to prevent their CPU load to affect each other.
+
+Depending on the setup the recording might also need to access the STUN server and/or the TURN server, although typically it will not be needed (specially if both the HPB and the recording server can access each other in the same internal network).
```

Comment:
```suggestion
Depending on the setup the recording might also need to access the STUN server and/or the TURN server, although typically it will not be needed (especially if both the HPB and the recording server can access each other in the same internal network).
```